notebooks/phase_evolution.ipynb

```diff
@@ -110,9 +110,220 @@
   },
   {
    "cell_type": "code",
-   "execution_count": null,
+   "execution_count": 4,
    "metadata": {},
-   "outputs": [],
+   "outputs": [
+    {
+     "data": {
```

Comment:
To avoid future conflicts, it might be better to clear the cells and add the unrendered notebook.